packages/loader/container-definitions/src/runtime.ts

```diff
@@ -135,6 +140,7 @@ export interface IContainerContext extends IDisposable {
     readonly serviceConfiguration: IClientConfiguration | undefined;
     readonly version: string;
     readonly previousRuntimeState: IRuntimeState;
+    pendingLocalState?: unknown;
```

Comment:
it should be readonly property

Reply:
Right now it's deleted on "connected" event and used to tell if we are in the process of rebasing ops.